### PR TITLE
Buttons styling

### PR DESCRIPTION
**Pull request by nastiche**

### CODE REVIEW

File: components/AiTaskInputForm/index.js [hunk 1]
```diff
@@ -98,6 +151,8 @@ export default function AiTaskInputForm({
 // Styled components for the form layout
```
Comment (ryliecc) [inline components/AiTaskInputForm/index.js:151]: Leaving comments inside your code when merging is bad practice. You can leave it in for now, but keep it in mind at the end of your project :)

File: components/NavigationLink/NavigationLink.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import styled, { css } from "styled-components";
+
+export const NavigationLinkWrapper = styled.span`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 100%;
+  width: 50px;
```
Comment (ryliecc) [inline components/NavigationLink/NavigationLink.js:8]: Just a suggestion, but you could replace px with a more responsive unit like em and rem. If you want to make sure you keep this exact styling you can use [this tool](https://nekocalc.com/de/px-zu-rem-umrechner) to convert everything. :)

**Review by ryliecc — APPROVED** [review]
LGTM 🥳

File: components/AiTaskInputForm/index.js (hunk 1 above)
Comment (nastiche) [reply components/AiTaskInputForm/index.js:151]: Which comments do you mean?

File: components/NavigationLink/NavigationLink.js (hunk 2 above)
Comment (nastiche) [reply components/NavigationLink/NavigationLink.js:8]: Will do , tnx!

**Pull request merged**